store/sqlstore/draft_store.go

```diff
@@ -49,17 +50,31 @@ func newSqlDraftStore(sqlStore *SqlStore, metrics einterfaces.MetricsInterface)
 	}
 }
 
-func (s *SqlDraftStore) Get(userId, channelId, rootId string) (*model.Draft, error) {
+func (s *SqlDraftStore) Get(userId, channelId, rootId string, includeDeleted bool) (*model.Draft, error) {
 	query := s.getQueryBuilder().
-		Select("*").
+		Select(
```

Comment:
Any specific reason to choose column names instead of `*`, also if not *, you could use `draftSliceColumns`?